### PR TITLE
Add backbone warning

### PR DESCRIPTION
**Pull request by devmanhinton**
@yaymukund 

Docs for backbone Dependency
1. Note at top of page
2. Link to Section
3. Section on Depdencies

### CODE REVIEW

**Comment by yaymukund** [comment]
There are still some typos... funtionality, for example.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ EndDash
 
 EndDash is a bindings-aware client-side templating language built on top of valid HTML.
 
+At this point EndDash relies on Backbone or Backbone style objects for full funtionality.
+Please see the depndency section for further details.
```
Comment (yaymukund) [inline README.md]: You can do:

``` md
Please [see the dependency section](#dependencies) for further details.
```

Renders like this:

Please [see the dependency section](#dependencies) for further details.

Comment (devmanhinton) [inline README.md]: Ahhh yes, I meant to that. Good catch :)

Comment (devmanhinton) [inline README.md]: Addressed and update to this.

**Comment by devmanhinton** [comment]
@yaymukund 'some'? That is the only typo that I saw (now fixed). I also pasted into google docs and got no other misspellings. Please let me know if you notice any other typos and I will fix them. I do not see any others.

**Comment by yaymukund** [comment]
"dependency" was a typo but you fixed it when you changed the link.

**Pull request merged**